src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolution_ArgsToParameters.cs

```diff
@@ -114,29 +113,38 @@ public ImmutableArray<int> ToImmutableArray()
             ParameterMap argsToParameters = new ParameterMap(parametersPositions, argumentCount);
 
             // We have analyzed every argument and tried to make it correspond to a particular parameter. 
-            // There are now four questions we must answer:
+            // We must now answer the following questions:
             //
-            // (1) Is there any named argument used out-of-position and followed by unnamed arguments?
-            // (2) Is there any argument without a corresponding parameter?
-            // (3) Was there any named argument that specified a parameter that was already
+            // (1) Is there any named argument that were specified twice?
```

Comment:
>Is there any named argument that were specified twice? [](start = 19, length = 54)

It feels like we shouldn't start with this check. Consider adding it last. Usually, adding new checks last is a better approach, especially for condition that was not considered an error before.